community/dnscrypt-proxy/APKBUILD

```diff
@@ -36,9 +36,10 @@ build() {
 
 package() {
 	cd "$builddir"/"$pkgname"
-	mkdir -p "$pkgdir"/etc/"$pkgname"
-	mkdir -p "$pkgdir"/var/log/"$pkgname"
-	mkdir -p "$pkgdir"/usr/share/"$pkgname"
+	install -d -o $pkgusers -g $pkggroups -m 755 "$pkgdir"/etc/$pkgname
+        install -d -o $pkgusers -g $pkggroups -m 755 "$pkgdir"/usr/share/$pkgname
+        install -d -o $pkgusers -g $pkggroups -m 755 "$pkgdir"/var/cache/$pkgname
+        install -d -o $pkgusers -g $pkggroups -m 755 "$pkgdir"/var/log/$pkgname
```

Comment:
why there's different indent?

Reply:
@ianbashford please fix the indendation and will merge it. Thanks!

Reply:
Doh!  Mixed tabs and spaces which didn't show up in my vim config (which I'll change right now)
This commit should be clean now